specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2019-12-01-preview/definitions.json

```diff
@@ -1784,9 +1784,63 @@
           "$ref": "#/definitions/PipelineDiagnosticSettings",
           "description": "Diagnostic settings for incoming/outgoing HTTP messages to the Backend"
         },
-        "enableHttpCorrelationHeaders": {
```

Comment:
>"enableHttpCorrelationHeaders" [](start = 8, length = 30)

did you mean to remove this from the -preview api-version?

Reply:
yes, it is meant to be removed in the new api-version